SU-SWS/stanford_profile_helper
# D8CORE-2285: adding the css to the pages that need the ckeditor style…

…s and made styles less specific to paragraphs

# READY FOR REVIEW

# Summary
- Made the ckeditor styles less specific
- Included them with the react paragraphs folder so pages such as events get the styles

# Needed By (Date)
- 10/16

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Test the editing and pages for a Basic, News, Events and People pages. 
4. Verify you can see the styles in the editor and the page. In particular, there was a problem with the card text. That on is easy to see.

# Affected Projects or Products
- Stanford Basic
- SOE
- Sites using the profile helper

# Associated Issues and/or People
- D8CORE-2258
- @katrialesser, Would this impact your work?

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
